src/Pages/contactUs/components/contactUs.scss

```diff
@@ -0,0 +1,197 @@
+.titleBlock > img {
+    width: 100%;
+    height: 100%;
+}
+
+.titleBlock {
+    position: relative;
+}
+
+.titleBlock > h1 {
+    color: white;
+    font-size: 48pt;
+    margin: 0;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.QuickLinksBlock {
+    margin-top: -2.2%;
+    background-color: #447399;
+    width: 100%;
+    height: 6cm;
+}
+
+.QuickLinksBlock > h2 {
+    padding-top: -5cm;
+    font-size: 24pt;
+    text-align: center;
+    color: white;
+    padding-top: 1%;
+}
+
+ul {
```

Comment:
lots of global css here, please wrap in the class name of the top dive in this component